sale_order_customer_free_ref/models/sale_order.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    client_order_ref = fields.Char(
+        compute="_compute_client_order_ref",
+        string="Customer Reference",
+        store=True,
+        copy=False,
+        readonly=True,
```

Comment:
shouldn't be `readonly=False` to preserve std behavior? :thinking:

Reply:
No the idea is `customer_order_number` and `customer_free_ref` are modified and `client_order_ref` is readonly.
Or that is what was requested.